apps/prairielearn/src/schemas/schemas/infoElementCourse.json

```diff
@@ -90,6 +90,32 @@
           }
         }
       }
+    },
+    "dynamicDependencies": {
+      "description": "The element's client-side dynamic dependencies.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "comment": {
+          "description": "Arbitrary comment for reference purposes.",
+          "anyOf": [{ "type": "string" }, { "type": "array" }, { "type": "object" }]
+        },
+        "nodeModulesScripts": {
+          "description": "The scripts required by this element from /node_modules as an importmap.",
+          "type": "object",
+          "additionalProperties": { "type": "string" }
+        },
```

Comment:
I consider it a mistake to have introduced `node_modules` support for course elements. Maybe we can avoid introducing this entirely?

Reply:
I don't fully agree with this. Note that some course elements are extensions/changes to core elements, so in many cases instructors may copy the content of the original element and make changes as necessary. In those cases, having a dependency that matches the ones in the equivalent core element probably makes sense. I understand not adding them in extensions, though.

Reply:
The problem is that we will inevitably update a dependency in such a way that it breaks people's forks of elements. I suppose we could keep this if we document that people accept the risk of breakages if they use it.

Reply:
Done in 4265f0a0fb443648ace64eeb7390f81e12790352.